### PR TITLE
Availability of Google Cloud data access logs in mart

### PR DESCRIPTION
**Pull request by SorenSpicknall**
# Description

In order to facilitate more detailed dashboarding for infra costs, this PR seeks to republish the entirety of `stg_audit__cloudaudit_googleapis_com_data_access` in mart, where it'll be usable by Metabase.

This contributes to, but does not yet fully resolve, #2022.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation
- [ ] agencies.yml

## How has this been tested?

At the time of writing, table creation has been tested in the staging project via locally issued dbt commands.

### CODE REVIEW

File: warehouse/models/mart/audit/fct_bigquery_data_access.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        job_type,
+        dbt_invocation_id,
+        create_disposition,
+        destination_table,
+        priority,
+        query,
+        statement_type,
+        write_disposition,
+        duration_in_seconds,
+        referenced_tables,
```
Comment (atvaccaro) [inline warehouse/models/mart/audit/fct_bigquery_data_access.sql:20]: can you either unnested in this model or create another table on top that unnests? I think seeing a rough breakdown of data volumes by queried table (i.e. read not write) may be useful; a table that is expensive to build but efficient to query is probably preferable for most of our uses cases.

**Review by atvaccaro — CHANGES_REQUESTED** [review]

**Review by atvaccaro — CHANGES_REQUESTED** [review]
I think even the unnested model seems like a fact? maybe `fct_bigquery_referenced_tables` and then you could add a caveat in the documentation about how it will overcount

**Comment by SorenSpicknall** [comment]
> I think even the unnested model seems like a fact? maybe `fct_bigquery_referenced_tables` and then you could add a caveat in the documentation about how it will overcount

Totally right there - I gave it that name before actually deciding how to structure the table. I'll make that change in the next commit.

**Review by atvaccaro — APPROVED** [review]
lgtm!

**Pull request merged**